File: C-programming/helloWorld.c
#include <stdio.h>

int main() {
    char s[100]; // Allocate memory for the string with a sufficient size
    printf("Hello World!\n");

    printf("Enter your full name: ");
    fgets(s, sizeof(s), stdin); // Read the entire line, including spaces

    printf("Your name is: %s", s);
    return 0;
}

File: TypeScript-Tutorial/helloWorld.ts
// Import readline module
import * as readline from 'readline';

// Create interface for input/output
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

console.log("Hello World!");

rl.question("Enter your full name: ", (name) => {
    console.log(`Your name is: ${name}`);
    rl.close(); // Close the interface
});
